app.config.yaml

```diff
@@ -26,6 +26,16 @@ application:
               require-adobe-auth: false
               raw-http: true
               final: true
+          remove-payment:
```

Comment:
It would be good to at least use `payment-method` instead of `payment`, since it's removing an available payment method from the list, not the payment itself. This also applies to the JS code and the action name.
Some suggestions:
- `filter-payment-method` (filter out)
- `exclude-payment-method`
- `remove-available-payment-method` (or `remove-payment-method`)

The code looks overall good, but I'd like to hear thoughts on this before merging: https://github.com/magento-commerce/oope-payment-methods/pull/17/files#r1985669650

Reply:
Additionally, I shared that I prefer `filter-payment-"method(s)"` (because another action `validate-payment` is to validate payment happened not method itself), but it's ok as long as we have clear documentation later . 
Thanks for checking!